src/main/kotlin/com/terraformation/backend/customer/api/OrganizationsController.kt

```diff
@@ -360,12 +362,14 @@ data class OrganizationPayload(
     val timeZone: ZoneId?,
     @Schema(
         description = "The total number of users in the organization, including the current user.")
+    val tfContactUserId: UserId?,
```

Comment:
This should go above the `@Schema` since the description there is for `totalUsers`.